dockerfiles/analytics/Dockerfile

```diff
@@ -53,30 +54,31 @@ RUN \
         curl \
         wget \
         unzip \
+        ca-certificates \
     && rm -rf /var/lib/apt/lists/*
 
+# create the non-root user and group and set MOTD login message
+RUN \
```

Comment:
@NishikaDeSilva let's move this `RUN` instruction ahead of the package installation instruction since, this instruction is bound to remain static, thus rebuilding the generated image layer is not required during an image rebuild.

Let's do the same with entrypoint file `COPY` instruction. The file is bound to change less often.

Please refer to latest product version Dockerfiles for examples.